Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,60 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var entry = 1;
+            var maxEntries = 3;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'maxEntries'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832006)